official/recommendation/neumf_model.py

```diff
@@ -381,8 +385,9 @@ def compute_eval_loss_and_metrics_helper(logits,              # type: tf.Tensor
   # ignore padded examples
   example_weights *= tf.cast(expanded_metric_weights, tf.float32)
 
-  cross_entropy = tf.losses.sparse_softmax_cross_entropy(
-      logits=softmax_logits, labels=eval_labels, weights=example_weights)
+  cce = tf.keras.losses.SparseCategoricalCrossentropy(from_logits=True)
```

Comment:
is this in the estimator code path or the new keras path? if former, i think maybe leave it as tf.compat.v1.losses.sparse_softmax_cross_entropy unless you plan to verify convergence with the old code path